package.json

```
@@ -13,7 +13,9 @@
     "migration:create": "bunx --bun drizzle-kit generate",
     "migration:drop": "bunx --bun drizzle-kit drop",
     "migration:run": "bunx --bun drizzle-kit migrate",
-    "migration:snapshots": "bunx --bun drizzle-kit up"
+    "migration:snapshots": "bunx --bun drizzle-kit up",
```

Why are these using bunx? How are we calling these scripts? Are we using `bun run migration:snapshots`? If so then you're using Bun to call bun and being rebundent
```suggestion
    "migration:snapshots": "drizzle-kit up",
```

---

Running via 'bun migration:run' for example, this style is not new to the change and was copied over from another project. I will resolve this for now, but if you have a strong opinion please open an issue for the style change and @nobrayner can make a call and I can do that as it's own change.